behat.yml

```diff
@@ -0,0 +1,4 @@
+default:
```

Comment:
This file is not needed, the `wp-cli/wp-cli-tests` framework takes care of this, you just need to run `composer behat` instead of `behat` or `vendor/bin/behat`.

The main issue here is that this hard-coded path is not always available anyway, depending on how you install the command.

Reply:
@schlessera Removed it. Tests are not running for Travis, is there anything else that needs to be done at code level? Should I send a PR to add this repo to the bundle?

Reply:
I haven't hooked it up to Travis yet. Let me do that...

Reply:
Ok, Travis should be active now, let's see whether it picks up on the PR after the next commit...